cfpb/consumerfinance.gov
# Carousel: Adjust thumbnail focus state

## Changes

- Remove `o-carousel_thumbnail-selected` class from mobile carousel links.
- Remove container `div` from mobile thumbnail links.
- Move carousel image layering behind thumbnails.


## How to test this PR

1. Pull branch, run `gulp clean && gulp build` visit homepage and tab through the carousel thumbnails. The current focus rectangle should be on top of everything else.


## Screenshots

Before:
<img width="377" alt="Screen Shot 2020-12-21 at 12 11 36 PM" src="https://user-images.githubusercontent.com/704760/102804203-6b03f700-4387-11eb-911c-efa7b0dab0c8.png">

After:
<img width="377" alt="Screen Shot 2020-12-21 at 12 11 28 PM" src="https://user-images.githubusercontent.com/704760/102804213-6f301480-4387-11eb-9201-a73293d23e68.png">


Before:
<img width="331" alt="Screen Shot 2020-12-21 at 11 24 54 AM" src="https://user-images.githubusercontent.com/704760/102804225-7525f580-4387-11eb-963f-282b8f35dc5f.png">

After:
<img width="323" alt="Screen Shot 2020-12-21 at 11 25 01 AM" src="https://user-images.githubusercontent.com/704760/102804241-79eaa980-4387-11eb-8046-d7a466bb57f3.png">
